test/__snapshots__/runAll.spec.js.snap

```diff
@@ -81,35 +81,6 @@ exports[`runAll should skip stashing changes if no lint-staged files are changed
 LOG No staged files match any of provided globs."
 `;
 
-exports[`runAll should skip updating stash if there are errors during linting 1`] = `
```

Comment:
Hmm why is this snapshot got removed?

Reply:
Hmm I’m not sure about this. Looking at the whole snap file, it seems to me the snapshot _is_ there:

https://github.com/iiroj/lint-staged/blob/4396e2b951d2e384767e90559d9cd7086d2a3401/test/__snapshots__/runAll.spec.js.snap#L84

I think in the jest diff it only added some tabs in the middle of the output, but for some reason the git diff shows it’s entirely deleted.